doc/how_to/custom_components/reactive_esm/reactive_esm_layout.md

```diff
@@ -0,0 +1,143 @@
+# Create Layouts With ReactiveESM
+
+In this guide we will show you how to build custom layouts using HTML and `ReactiveESM`.
+
+## Layout a single Panel Component
+
+You can layout a single Panel component as follows.
+
+:::::{tab-set}
+
+::::{tab-item} JavaScript
+
+```{pyodide}
+import param
+import panel as pn
+
+pn.extension()
+
+class LayoutSingleObject(pn.ReactiveESM):
+    object = param.ClassSelector(class_=pn.viewable.Viewable, allow_refs=False)
```

Comment:
Why do I need `allow_refs=True` in this example? Is that really nescessary?

Reply:
No, `ReactiveESM` probably shouldn't have the `allow_refs=True` by default.

Reply:
But I would also like to be able to use references in my `ReactiveESM` layouts.

Reply:
You can you just have to enable it manually.